topics/benchmarks.md

```diff
@@ -13,24 +13,34 @@ The following options are supported:
      -p <port>          Server port (default 6379)
      -s <socket>        Server socket (overrides host and port)
      -a <password>      Password for Redis Auth
+     --user <username>  Used to send ACL style 'AUTH username pass'. Needs -a.
      -c <clients>       Number of parallel connections (default 50)
      -n <requests>      Total number of requests (default 100000)
-     -d <size>          Data size of SET/GET value in bytes (default 2)
+     -d <size>          Data size of SET/GET value in bytes (default 3)
      --dbnum <db>       SELECT the specified db number (default 0)
+     --threads <num>    Enable multi-thread mode.
+     --cluster          Enable cluster mode.
+     --enable-tracking  Send CLIENT TRACKING on before starting benchmark.
      -k <boolean>       1=keep alive 0=reconnect (default 1)
-     -r <keyspacelen>   Use random keys for SET/GET/INCR, random values for SADD
-      Using this option the benchmark will expand the string __rand_int__
-      inside an argument with a 12 digits number in the specified range
-      from 0 to keyspacelen-1. The substitution changes every time a command
-      is executed. Default tests use this to hit random keys in the
-      specified range.
+     -r <keyspacelen>   Use random keys for SET/GET/INCR, random values for SADD,
+                         random members and scores for ZADD.
+     Using this option the benchmark will expand the string __rand_int__
+     inside an argument with a 12 digits number in the specified range
+     from 0 to keyspacelen-1. The substitution changes every time a command
+     is executed. Default tests use this to hit random keys in the
+     specified range.
```

Comment:
Alignement (since this is a code block):

```suggestion
     -r <keyspacelen>    Use random keys for SET/GET/INCR, random values for
                         SADD, random members and scores for ZADD.
                         Using this option the benchmark will expand the string
                         __rand_int__ inside an argument with a 12 digits number
                         in the specified range from 0 to keyspacelen-1. The
                         substitution changes every time a command is executed.
                         Default tests use this to hit random keys in the
                         specified range.
```